internal/buildscripts/packaging/installer/install.ps1

```diff
@@ -375,6 +405,10 @@ if ($with_fluentd -And (Test-Path -Path "$fluentd_base_dir\bin\fluentd")) {
     throw "$fluentd_base_dir\bin\fluentd is already installed. Remove/Uninstall fluentd and re-run this script."
 }
 
+if ($with_dotnet_tracing -And (Test-Path -Path "$dotnet_tracing_base_dir\SignalFx.Tracing.ClrProfiler.Native.dll")) {
+    throw "SignalFx .NET tracing is already installed. Remove/Uninstall SignalFx .NET tracing and re-run this script."
```

Comment:
It seems worth to automatically run the uninstall or at least give an option to do it.

Reply:
I think this is a fine idea, but we're not doing that for the other installers in this script. What do you think about making that a separate effort?

Reply:
I think this kind of logic should be in the PS module actually. 
Currently `$dotnet_tracing_install_dir` `$dotnet_tracing_base_dir` `SignalFx.Tracing.ClrProfiler.Native.dll` values are hardcoded into the collector's script, but these can change / be specific to the version. 

Should do something like > 
```powershell
if ($with_dotnet_instrumentation -And (Test-Path -Path Get-SignalFxInstallDirectory))
```

Reply:
I have made this change. However I just realized that the tracing/profiling installer runs after other things have been installed, whereas before, the test for whether anything has already been installed happened before anything got installed, so we had the ability to abort installing anything unless we knew that everything was going to succeed. Perhaps the tracing module should export another function (an is_installed test) that we can call early on in the process, before we've installed anything.

Reply:
Linking this chance here also > 
https://github.com/signalfx/signalfx-dotnet-tracing/pull/653

Reply:
@pmcollins I updated the module, it's now merged.

This should work now > 

```csharp
if ($with_dotnet_instrumentation -and Get-IsSignalFxInstalled)
```